features/org.eclipse.equinox.executable.feature/library/org.eclipse.launcher.tests/src/org/eclipse/equinox/launcher/TestLauncherApp.java

```diff
@@ -11,7 +11,7 @@
  * Contributors:
  *     Umair Sair - initial API and implementation
  *******************************************************************************/
-package main;
+package org.eclipse.equinox.launcher;
```

Comment:
The JNIBridge import below should be removed if we move to this package.

Reply:
Done.